Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
UpdateCTestConfiguration  from :/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/DartConfiguration.tcl
Test project /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 1
      Start  1: incorrect_solution_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
Looked in the following places:
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Release/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Release/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Debug/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Debug/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/MinSizeRel/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/MinSizeRel/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/RelWithDebInfo/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/RelWithDebInfo/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Deployment/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Deployment/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Development/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Development/incorrect_solution_tests

1: Test command: 
1: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
 1/11 Test  #1: incorrect_solution_tests .........***Not Run   0.00 sec
test 2
      Start  2: model1_tests

2: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model1_tests
2: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
2: Test timeout computed to be: 5
2: [==========] Running 7 tests from 1 test suite.
2: [----------] Global test environment set-up.
2: [----------] 7 tests from DijkstraTestDijkstraTest
2: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
2: Expected equality of these values:
2:   distances[1]
2:     Which is: -52
2:   2
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:27: Failure
2: Expected equality of these values:
2:   distances[2]
2:     Which is: -50
2:   4
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
2: Expected equality of these values:
2:   distances[3]
2:     Which is: -58
2:   1
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:29: Failure
2: Expected equality of these values:
2:   distances[4]
2:     Which is: -40
2:   14
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
2: Expected equality of these values:
2:   distances[5]
2:     Which is: -52
2:   7
2: 
2: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
2: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
2: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
2: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
2: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
2: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:90: Failure
2: Expected equality of these values:
2:   distances[2]
2:     Which is: -6
2:   3
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
2: Expected equality of these values:
2:   distances[3]
2:     Which is: -5
2:   3
2: 
2: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
2: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:105: Failure
2: Expected equality of these values:
2:   distances[3]
2:     Which is: -1
2:   numeric_limits<int>::max()
2:     Which is: 2147483647
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:106: Failure
2: Expected equality of these values:
2:   distances[4]
2:     Which is: -1
2:   numeric_limits<int>::max()
2:     Which is: 2147483647
2: 
2: [  FAILED  ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
2: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:118: Failure
2: Expected equality of these values:
2:   distances[0]
2:     Which is: -18
2:   0
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
2: Expected equality of these values:
2:   distances[1]
2:     Which is: -16
2:   -1
2: 
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
2: Expected equality of these values:
2:   distances[2]
2:     Which is: -17
2:   -2
2: 
2: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
2: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 2/11 Test  #2: model1_tests .....................***Exception: SegFault  0.46 sec
test 3
      Start  3: model10_tests

3: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model10_tests
3: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
3: Test timeout computed to be: 5
3: [==========] Running 7 tests from 1 test suite.
3: [----------] Global test environment set-up.
3: [----------] 7 tests from DijkstraTestDijkstraTest
3: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
3: Expected equality of these values:
3:   distances[1]
3:     Which is: -12
3:   2
3: 
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:27: Failure
3: Expected equality of these values:
3:   distances[2]
3:     Which is: -10
3:   4
3: 
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
3: Expected equality of these values:
3:   distances[3]
3:     Which is: -18
3:   1
3: 
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:29: Failure
3: Expected equality of these values:
3:   distances[4]
3:     Which is: 0
3:   14
3: 
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
3: Expected equality of these values:
3:   distances[5]
3:     Which is: -12
3:   7
3: 
3: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
3: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
3: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
3: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
3: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
3: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:90: Failure
3: Expected equality of these values:
3:   distances[2]
3:     Which is: -6
3:   3
3: 
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
3: Expected equality of these values:
3:   distances[3]
3:     Which is: -5
3:   3
3: 
3: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
3: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
3: [       OK ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
3: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:118: Failure
3: Expected equality of these values:
3:   distances[0]
3:     Which is: -6
3:   0
3: 
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
3: Expected equality of these values:
3:   distances[1]
3:     Which is: -4
3:   -1
3: 
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
3: Expected equality of these values:
3:   distances[2]
3:     Which is: -5
3:   -2
3: 
3: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
3: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 3/11 Test  #3: model10_tests ....................***Exception: SegFault  0.34 sec
test 4
      Start  4: model2_tests

4: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model2_tests
4: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
4: Test timeout computed to be: 5
4: [==========] Running 7 tests from 1 test suite.
4: [----------] Global test environment set-up.
4: [----------] 7 tests from DijkstraTestDijkstraTest
4: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
4: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
4: Expected equality of these values:
4:   distances[1]
4:     Which is: -4
4:   2
4: 
4: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:29: Failure
4: Expected equality of these values:
4:   distances[4]
4:     Which is: 0
4:   14
4: 
4: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
4: Expected equality of these values:
4:   distances[5]
4:     Which is: -8
4:   7
4: 
4: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
4: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
4: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
4: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
4: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
4: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
4: [       OK ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
4: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
4: [       OK ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
4: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
4: [       OK ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
4: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 4/11 Test  #4: model2_tests .....................***Exception: SegFault  0.35 sec
test 5
      Start  5: model3_tests

5: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model3_tests
5: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
5: Test timeout computed to be: 5
5: [==========] Running 7 tests from 1 test suite.
5: [----------] Global test environment set-up.
5: [----------] 7 tests from DijkstraTestDijkstraTest
5: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
5: Expected equality of these values:
5:   distances[1]
5:     Which is: 0
5:   2
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:27: Failure
5: Expected equality of these values:
5:   distances[2]
5:     Which is: 0
5:   4
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
5: Expected equality of these values:
5:   distances[3]
5:     Which is: 0
5:   1
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:29: Failure
5: Expected equality of these values:
5:   distances[4]
5:     Which is: 10
5:   14
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
5: Expected equality of these values:
5:   distances[5]
5:     Which is: 6
5:   7
5: 
5: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
5: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
5: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
5: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
5: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
5: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:89: Failure
5: Expected equality of these values:
5:   distances[1]
5:     Which is: 0
5:   -4
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:90: Failure
5: Expected equality of these values:
5:   distances[2]
5:     Which is: 0
5:   3
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
5: Expected equality of these values:
5:   distances[3]
5:     Which is: 1
5:   3
5: 
5: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
5: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:105: Failure
5: Expected equality of these values:
5:   distances[3]
5:     Which is: -1
5:   numeric_limits<int>::max()
5:     Which is: 2147483647
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:106: Failure
5: Expected equality of these values:
5:   distances[4]
5:     Which is: -1
5:   numeric_limits<int>::max()
5:     Which is: 2147483647
5: 
5: [  FAILED  ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
5: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
5: Expected equality of these values:
5:   distances[1]
5:     Which is: 0
5:   -1
5: 
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
5: Expected equality of these values:
5:   distances[2]
5:     Which is: 0
5:   -2
5: 
5: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
5: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 5/11 Test  #5: model3_tests .....................***Exception: SegFault  0.36 sec
test 6
      Start  6: model4_tests

6: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model4_tests
6: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
6: Test timeout computed to be: 5
6: [==========] Running 7 tests from 1 test suite.
6: [----------] Global test environment set-up.
6: [----------] 7 tests from DijkstraTestDijkstraTest
6: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
6: Expected equality of these values:
6:   distances[1]
6:     Which is: -52
6:   2
6: 
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:27: Failure
6: Expected equality of these values:
6:   distances[2]
6:     Which is: -50
6:   4
6: 
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
6: Expected equality of these values:
6:   distances[3]
6:     Which is: -58
6:   1
6: 
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:29: Failure
6: Expected equality of these values:
6:   distances[4]
6:     Which is: -40
6:   14
6: 
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
6: Expected equality of these values:
6:   distances[5]
6:     Which is: -52
6:   7
6: 
6: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
6: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
6: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
6: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
6: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
6: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:90: Failure
6: Expected equality of these values:
6:   distances[2]
6:     Which is: -6
6:   3
6: 
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
6: Expected equality of these values:
6:   distances[3]
6:     Which is: -5
6:   3
6: 
6: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
6: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
6: [       OK ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
6: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:118: Failure
6: Expected equality of these values:
6:   distances[0]
6:     Which is: -18
6:   0
6: 
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
6: Expected equality of these values:
6:   distances[1]
6:     Which is: -16
6:   -1
6: 
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
6: Expected equality of these values:
6:   distances[2]
6:     Which is: -17
6:   -2
6: 
6: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
6: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 6/11 Test  #6: model4_tests .....................***Exception: SegFault  0.36 sec
test 7
      Start  7: model5_tests

7: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model5_tests
7: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
7: Test timeout computed to be: 5
7: [==========] Running 7 tests from 1 test suite.
7: [----------] Global test environment set-up.
7: [----------] 7 tests from DijkstraTestDijkstraTest
7: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
7: Expected equality of these values:
7:   distances[1]
7:     Which is: -48
7:   2
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:27: Failure
7: Expected equality of these values:
7:   distances[2]
7:     Which is: -50
7:   4
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
7: Expected equality of these values:
7:   distances[3]
7:     Which is: -58
7:   1
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:29: Failure
7: Expected equality of these values:
7:   distances[4]
7:     Which is: -40
7:   14
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
7: Expected equality of these values:
7:   distances[5]
7:     Which is: -52
7:   7
7: 
7: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
7: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
7: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
7: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
7: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
7: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:90: Failure
7: Expected equality of these values:
7:   distances[2]
7:     Which is: -6
7:   3
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
7: Expected equality of these values:
7:   distances[3]
7:     Which is: -5
7:   3
7: 
7: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
7: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:105: Failure
7: Expected equality of these values:
7:   distances[3]
7:     Which is: -1
7:   numeric_limits<int>::max()
7:     Which is: 2147483647
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:106: Failure
7: Expected equality of these values:
7:   distances[4]
7:     Which is: -1
7:   numeric_limits<int>::max()
7:     Which is: 2147483647
7: 
7: [  FAILED  ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
7: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:118: Failure
7: Expected equality of these values:
7:   distances[0]
7:     Which is: -15
7:   0
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
7: Expected equality of these values:
7:   distances[1]
7:     Which is: -16
7:   -1
7: 
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
7: Expected equality of these values:
7:   distances[2]
7:     Which is: -17
7:   -2
7: 
7: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
7: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 7/11 Test  #7: model5_tests .....................***Exception: SegFault  0.44 sec
test 8
      Start  8: model6_tests

8: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model6_tests
8: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
8: Test timeout computed to be: 5
8: [==========] Running 7 tests from 1 test suite.
8: [----------] Global test environment set-up.
8: [----------] 7 tests from DijkstraTestDijkstraTest
8: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
8: Expected equality of these values:
8:   distances[1]
8:     Which is: 0
8:   2
8: 
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
8: Expected equality of these values:
8:   distances[3]
8:     Which is: 0
8:   1
8: 
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
8: Expected equality of these values:
8:   distances[5]
8:     Which is: 0
8:   7
8: 
8: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
8: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
8: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
8: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
8: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
8: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:89: Failure
8: Expected equality of these values:
8:   distances[1]
8:     Which is: 0
8:   -4
8: 
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
8: Expected equality of these values:
8:   distances[3]
8:     Which is: 4
8:   3
8: 
8: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
8: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
8: [       OK ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
8: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
8: Expected equality of these values:
8:   distances[1]
8:     Which is: 0
8:   -1
8: 
8: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
8: Expected equality of these values:
8:   distances[2]
8:     Which is: 2147483647
8:   -2
8: 
8: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
8: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 8/11 Test  #8: model6_tests .....................***Exception: SegFault  0.41 sec
test 9
      Start  9: model7_tests

9: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model7_tests
9: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
9: Test timeout computed to be: 5
9: [==========] Running 7 tests from 1 test suite.
9: [----------] Global test environment set-up.
9: [----------] 7 tests from DijkstraTestDijkstraTest
9: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
9: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
9: Expected equality of these values:
9:   distances[1]
9:     Which is: -2147483648
9:   2
9: 
9: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
9: Expected equality of these values:
9:   distances[3]
9:     Which is: -2147483648
9:   1
9: 
9: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
9: Expected equality of these values:
9:   distances[5]
9:     Which is: -2147483648
9:   7
9: 
9: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
9: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
9: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
9: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
9: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
9: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
9: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:89: Failure
9: Expected equality of these values:
9:   distances[1]
9:     Which is: -2147483648
9:   -4
9: 
9: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
9: Expected equality of these values:
9:   distances[3]
9:     Which is: 4
9:   3
9: 
9: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
9: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
9: [       OK ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
9: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
9: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
9: Expected equality of these values:
9:   distances[1]
9:     Which is: -2147483648
9:   -1
9: 
9: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
9: Expected equality of these values:
9:   distances[2]
9:     Which is: 2147483647
9:   -2
9: 
9: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
9: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
 9/11 Test  #9: model7_tests .....................***Exception: SegFault  0.72 sec
test 10
      Start 10: model8_tests

10: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model8_tests
10: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
10: Test timeout computed to be: 5
10: [==========] Running 7 tests from 1 test suite.
10: [----------] Global test environment set-up.
10: [----------] 7 tests from DijkstraTestDijkstraTest
10: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
10: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
10: Expected equality of these values:
10:   distances[1]
10:     Which is: 2147483647
10:   2
10: 
10: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
10: Expected equality of these values:
10:   distances[3]
10:     Which is: 2147483647
10:   1
10: 
10: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:30: Failure
10: Expected equality of these values:
10:   distances[5]
10:     Which is: 2147483647
10:   7
10: 
10: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
10: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
10: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
10: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
10: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
10: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
10: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:89: Failure
10: Expected equality of these values:
10:   distances[1]
10:     Which is: 2147483647
10:   -4
10: 
10: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
10: Expected equality of these values:
10:   distances[3]
10:     Which is: 4
10:   3
10: 
10: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
10: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
10: [       OK ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
10: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
10: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
10: Expected equality of these values:
10:   distances[1]
10:     Which is: 2147483647
10:   -1
10: 
10: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
10: Expected equality of these values:
10:   distances[2]
10:     Which is: 2147483647
10:   -2
10: 
10: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
10: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
10/11 Test #10: model8_tests .....................***Exception: SegFault  0.37 sec
test 11
      Start 11: model9_tests

11: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/model9_tests
11: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
11: Test timeout computed to be: 5
11: [==========] Running 7 tests from 1 test suite.
11: [----------] Global test environment set-up.
11: [----------] 7 tests from DijkstraTestDijkstraTest
11: [ RUN      ] DijkstraTestDijkstraTest.MixedGraphTest
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:25: Failure
11: Expected equality of these values:
11:   distances[0]
11:     Which is: 14
11:   0
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:26: Failure
11: Expected equality of these values:
11:   distances[1]
11:     Which is: 6
11:   2
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:27: Failure
11: Expected equality of these values:
11:   distances[2]
11:     Which is: 8
11:   4
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:28: Failure
11: Expected equality of these values:
11:   distances[3]
11:     Which is: 0
11:   1
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:29: Failure
11: Expected equality of these values:
11:   distances[4]
11:     Which is: 18
11:   14
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cppErrors while running CTest
Output from these tests are in: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
:30: Failure
11: Expected equality of these values:
11:   distances[5]
11:     Which is: 6
11:   7
11: 
11: [  FAILED  ] DijkstraTestDijkstraTest.MixedGraphTest (0 ms)
11: [ RUN      ] DijkstraTestDijkstraTest.PositiveGraphTest
11: [       OK ] DijkstraTestDijkstraTest.PositiveGraphTest (0 ms)
11: [ RUN      ] DijkstraTestDijkstraTest.HandlesPositiveEdges
11: [       OK ] DijkstraTestDijkstraTest.HandlesPositiveEdges (0 ms)
11: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:88: Failure
11: Expected equality of these values:
11:   distances[0]
11:     Which is: 6
11:   0
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:89: Failure
11: Expected equality of these values:
11:   distances[1]
11:     Which is: 2
11:   -4
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:90: Failure
11: Expected equality of these values:
11:   distances[2]
11:     Which is: 0
11:   3
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:91: Failure
11: Expected equality of these values:
11:   distances[3]
11:     Which is: 1
11:   3
11: 
11: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeEdgesClosestToZero (0 ms)
11: [ RUN      ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph
11: [       OK ] DijkstraTestDijkstraTest.HandlesDisconnectedGraph (0 ms)
11: [ RUN      ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:119: Failure
11: Expected equality of these values:
11:   distances[1]
11:     Which is: 2
11:   -1
11: 
11: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/test.cpp:120: Failure
11: Expected equality of these values:
11:   distances[2]
11:     Which is: 1
11:   -2
11: 
11: [  FAILED  ] DijkstraTestDijkstraTest.HandlesNegativeCyclePrevention (0 ms)
11: [ RUN      ] DijkstraTestDijkstraTest.EmptyGraphTest
11/11 Test #11: model9_tests .....................***Exception: SegFault  0.43 sec

0% tests passed, 11 tests failed out of 11

Total Test time (real) =   4.23 sec

The following tests FAILED:
	  1 - incorrect_solution_tests (Not Run)
	  2 - model1_tests (SEGFAULT)
	  3 - model10_tests (SEGFAULT)
	  4 - model2_tests (SEGFAULT)
	  5 - model3_tests (SEGFAULT)
	  6 - model4_tests (SEGFAULT)
	  7 - model5_tests (SEGFAULT)
	  8 - model6_tests (SEGFAULT)
	  9 - model7_tests (SEGFAULT)
	 10 - model8_tests (SEGFAULT)
	 11 - model9_tests (SEGFAULT)
Test project /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build
      Start  1: incorrect_solution_tests
Could not find executable /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
Looked in the following places:
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Release/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Release/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Debug/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Debug/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/MinSizeRel/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/MinSizeRel/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/RelWithDebInfo/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/RelWithDebInfo/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Deployment/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Deployment/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Development/incorrect_solution_tests
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Development/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Release/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Release/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Debug/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Debug/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/MinSizeRel/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/MinSizeRel/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/RelWithDebInfo/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/RelWithDebInfo/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Deployment/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Deployment/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Development/incorrect_solution_tests
Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Development/incorrect_solution_tests
Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/incorrect_solution_tests
 1/11 Test  #1: incorrect_solution_tests .........***Not Run   0.00 sec
      Start  2: model1_tests
 2/11 Test  #2: model1_tests .....................***Exception: SegFault  0.01 sec
      Start  3: model10_tests
 3/11 Test  #3: model10_tests ....................***Exception: SegFault  0.01 sec
      Start  4: model2_tests
 4/11 Test  #4: model2_tests .....................***Exception: SegFault  0.01 sec
      Start  5: model3_tests
 5/11 Test  #5: model3_tests .....................***Exception: SegFault  0.01 sec
      Start  6: model4_tests
 6/11 Test  #6: model4_tests .....................***Exception: SegFault  0.01 sec
      Start  7: model5_tests
 7/11 Test  #7: model5_tests .....................***Exception: SegFault  0.01 sec
      Start  8: model6_tests
 8/11 Test  #8: model6_tests .....................***Exception: SegFault  0.01 sec
      Start  9: model7_tests
 9/11 Test  #9: model7_tests .....................***Exception: SegFault  0.01 sec
      Start 10: model8_tests
10/11 Test #10: model8_tests .....................***Exception: SegFault  0.01 sec
      Start 11: model9_tests
11/11 Test #11: model9_tests .....................***Exception: SegFault  0.01 sec

0% tests passed, 11 tests failed out of 11

Total Test time (real) =   0.13 sec

The following tests FAILED:
	  1 - incorrect_solution_tests (Not Run)
	  2 - model1_tests (SEGFAULT)
	  3 - model10_tests (SEGFAULT)
	  4 - model2_tests (SEGFAULT)
	  5 - model3_tests (SEGFAULT)
	  6 - model4_tests (SEGFAULT)
	  7 - model5_tests (SEGFAULT)
	  8 - model6_tests (SEGFAULT)
	  9 - model7_tests (SEGFAULT)
	 10 - model8_tests (SEGFAULT)
	 11 - model9_tests (SEGFAULT)
Errors while running CTest
Output from these tests are in: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/621286/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
